test/unit/app/managers/test_HDCAManager.py

```diff
@@ -58,7 +58,7 @@ def testable_url_for(*a, **k):
     return f"(fake url): {a}, {k}"
```

Comment:
You can replace `testable_url_for` with `from galaxy.app_unittest_utils.galaxy_mock import mock_url_builder`